main.go

```diff
@@ -62,6 +63,10 @@ func run(ctx context.Context, ms *xmain.State) (err error) {
 	if err != nil {
 		return err
 	}
+	darkThemeFlag, err := ms.Opts.Int64("D2_D_THEME", "dark_theme", "", math.MaxInt64, "the diagram dark theme ID. When left unset only the theme will be applied")
```

Comment:
i don't get why it'd be maxint64, this also would prevent 32-bit architectures (https://github.com/terrastruct/d2/pull/395). if you want to use a sentinel, just use -1

Reply:
it's true that this would be a better solution, will change that

Reply:
changed